src/main/java/org/entando/kubernetes/controller/database/DbmsVendorConfig.java

```diff
@@ -45,74 +41,74 @@ public String buildConnectionString() {
             };
         }
     },
-    ORACLE("docker.io/store/oracle/database-enterprise:12.2.0.1", 1521, "sys", "/ORCL", "sqlplus sys/Oradoc_db1:${DB_SID}",
-            "org.hibernate.dialect.Oracle10gDialect") {
+    ORACLE("org.hibernate.dialect.Oracle10gDialect", 1521, "sys", "sqlplus sys/Oradoc_db1:${DB_SID}") {
         public JdbcConnectionStringBuilder getConnectionStringBuilder() {
             return new JdbcConnectionStringBuilder() {
                 public String buildConnectionString() {
                     return String.format("jdbc:oracle:thin:@//%s:%s/%s", this.getHost(), this.getPort(), this.getDatabase());
                 }
             };
         }
+    },
+    DERBY("org.apache.derby.jdbc.EmbeddedDriver", "agile", "agile") {
```

Comment:
This is the jdbc driver, but the expected parameter is the JPA/HIbernate dialect